### ISSUE TITLE
Problems with Rainbow Six Black Ops mod

### ISSUE THREAD

**Issue by Lrflx** [dege-diosg/dgVoodoo2#87]
Black Ops is a compilation of missions from all old R6 games based on Rogue Spear engine, as I understand. It could be downloaded for free here:
https://www.moddb.com/mods/rainbow-six-black-ops-20

First problem is that preformance drops whenever text appears on screen in-game. I had similar problem with Requiem Avenging Angel and there I managed to fix it by checking Disable8and16bitModes in CAT, but with Black Ops it didn't help, guess it's a different problem.

Also this game appears to be sensitive to forced resolution and aa, cause 2d elements get corrupted immediately. I expected these types of problems from forcing aa, but I thought forcing resolution should be more compatible and glitchless? Maybe it could be tweaked somewhat?

**Comment by dege-diosg** [comment]
Yes, 2D elements seems not to be compatible with MSAA or resolution upscale. I don't think it can be tweaked on the wrapper side. Why not set a large resolution inside the game instead? As far as I remember the HUD scales relatively well.

I can't test the game now, but I released a WIP version this week and it contains a fix for Urban Operations problems.
Doesn't it help in regard of the performance problem?

https://www.vogons.org/viewtopic.php?p=1008529#p1008529

**Comment by dege-diosg** [comment]
I installed the mod, but cannot launch the game. I run R6BOConfig.exe and then R6BO.exe but nothing happens, I get a log:

```
OPTIONS: Working Directory is: g:\Games\OldDX\Rainbow Six Black Ops
OPTIONS: Reading registry key Software\Red Storm Entertainment\Black Ops
GAME: Could not read options
```

Am I missing something?

**Comment by Lrflx** [comment]
Hmm, I don't remeber exactly all the steps I did when installing the game, but there should be a config folder in the game root with files for registry setup, also to change compatibility layers, maybe it'll help?
